apple/job/__main__.py

```diff
@@ -184,6 +191,23 @@ def get_request_lambda(f, *args, **kwargs):
         start_date=date_range.start,
         end_date=date_range.end,
     ),
+
+    # For weekly aggregated data:
+    FetchParams(
+        openpodcast_endpoint="episodes",
```

Comment:
TODO: should this endpoint be changed? E.g. `episodes_aggregated`?

Reply:
It might work like this because the grouping is part of the payload and can be differentiated on the API side.